docs/guide/getting-started.md

```diff
@@ -189,3 +189,32 @@ If you would like to use it, install it with `npm i -g nodemon`. Then you can ru
 ### Now you are ready to start developing!🎉 🎉 🎉 🎉 
 
 Next, feel free to read through other parts of the documentation and codebase that interest you, and assign yourself to an issue that you want to work on!
+
```

Comment:
Store this in a new MD file called contributing or something along those lines (maybe pull request guidelines?)

(This is because our getting-started.md is getting too crowded)

Reply:
Hmm, that is what I felt, too, at first, but then I wasn't sure if PR creation should be part of the introductory guide to get developers started.